### PR TITLE
Tier0intfip

### PR DESCRIPTION
**Pull request by enhaocui**

### CODE REVIEW

**Comment by enhaocui** [comment]
> Don't you want to add it to the documentation?

Sure, let me add it to documentation and add UT for this new field. Thanks!

File: website/docs/r/policy_bgp_neighbor.html.markdown [hunk 1]
```diff
@@ -59,7 +58,7 @@ The following arguments are supported:
 * `neighbor_address` - (Required) Neighbor IP Address.
 * `password` - (Optional) Password for BGP neighbor authentication. Set to the empty string to clear out the password.
 * `remote_as_num` - (Required) 4 Byte ASN of the neighbor in ASPLAIN Format.
-* `source_addresses` - (Optional) A list of up to 8 source IP Addresses for BGP peering.
+* `source_addresses` - (Optional) A list of up to 8 source IP Addresses for BGP peering. `ip_addresses` field of an existing `nsxt_policy_tier0_gateway` can be used here.
```
Comment (annakhm) [inline website/docs/r/policy_bgp_neighbor.html.markdown:61]: This should be nsxt_policy_tier0_gateway_interface

File: nsxt/resource_nsxt_policy_bgp_neighbor_test.go [hunk 2]
```diff
@@ -332,6 +358,7 @@ resource "nsxt_policy_bgp_neighbor" "test" {
   neighbor_address      = "%s"
   remote_as_num         = "%s"
   password              = "%s"
+  source_addresses = nsxt_policy_tier0_gateway_interface.test.ip_addresses
```
Comment (annakhm) [inline nsxt/resource_nsxt_policy_bgp_neighbor_test.go:361]: nit: please align to upper fields

Comment (enhaocui) [reply nsxt/resource_nsxt_policy_bgp_neighbor_test.go]: Done. Thanks.

File: website/docs/r/policy_bgp_neighbor.html.markdown (hunk 1 above)
Comment (enhaocui) [reply website/docs/r/policy_bgp_neighbor.html.markdown]: > This should be nsxt_policy_tier0_gateway_interface

Thanks for the catch! Done.

**Pull request merged**